youtube/generic.md

```diff
@@ -1,37 +1,46 @@
-...
+## Youtube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://www.youtube.com/eddiejaoude?sub_confirmation=1
 
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
+**Make sure to tell your friends about it, so that they also learn something new!**
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+## Youtube Shorts
+I have another youtube channel for short clips, please **take a moment subscribing it: https://www.youtube.com/eddiejaoudetv**
 
-Alslo join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at http://eddiehub.org
+
+Make sure to checkout my website and sign up to our community **newsletter**: http://eddiejaoude.io for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organisation**:
+
+- Go to https://github.com/EddieHubCommunity/support/issues/choose 
```

Comment:
```suggestion
- Go to https://github.com/EddieHubCommunity/support/issues/new/choose 
```
I think the `choose` route is under `new`
OR Maybe we can link to the `Invite Me to the org` template directly? -> https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization